packages/ra-ui-materialui/src/input/BooleanInput.js

```diff
@@ -24,14 +24,17 @@ export class BooleanInput extends Component {
             ...rest
         } = this.props;
 
+        const { value, ...inputProps } = input;
```

Comment:
this change seems unrelated, what's the motivation?

Reply:
Can be done in another PR. I stumbled upon this one while testing this feature. `BooleanInput` do not pass the input props (`name`, etc.) to the inner input element

Reply:
I found this while trying to set a `BooleanInput` value with Cypress.

Reply:
Should I open a PR on master for this ?

Reply:
nevermind, let's keep it here